### PR TITLE
New installation options for Quick Start section

### PR DESCRIPTION
**Pull request by alparslanavci**

### CODE REVIEW

**Review by Holmistr — COMMENTED** [review]
@alparslanavci Thinking whether we shouldn't rather use the code snippet tabs with "Brew", "RPM", "DEB" headlines to avoid noise. WDYT @Serdaro ?

**Comment by alparslanavci** [comment]
@Holmistr, updated as you mentioned.

File: src/docs/asciidoc/getting_started.adoc [hunk 1]
```diff
@@ -16,14 +16,33 @@ You can use the Hazelcast
...
+
+[source,bash,indent=0,subs="verbatim,attributes",role="secondary"]
+.RPM
+----
+$ wget https://bintray.com/hazelcast/rpm/rpm -O bintray-hazelcast-rpm.repo
+$ sudo mv bintray-hazelcast-rpm.repo /etc/yum.repos.d/
+$ sudo yum install hazelcast
+----
+
+After the installation of Hazelcast CLI, the package managers will display the installation directory and usage information.
```
Comment (Serdaro) [inline src/docs/asciidoc/getting_started.adoc:44]: Could we add a post-installation information example for one of the ways here?

Something similar to:

" ..... and usage information. The following is an example if you have used brew:

???
???
"

File: src/docs/asciidoc/getting_started.adoc [hunk 2]
```diff
@@ -16,14 +16,33 @@ You can use the Hazelcast
...
 to install and start Hazelcast IMDG.
 
-Assuming you have the link:https://brew.sh/[Homebrew^] package manager, run the following commands:
+Run the following commands to install Hazelcast CLI:
 
-[source,bash,subs="attributes+"]
+[source,bash,indent=0,subs="verbatim,attributes",role="secondary"]
+.Homebrew
 ----
 $ brew tap hazelcast/hz
```
Comment (Serdaro) [inline src/docs/asciidoc/getting_started.adoc:24]: Before the actual brew, Debian and RPM commands, don't we need to mention that the user already needs to have the related package manager installations? (maybe giving links to the respective manager, in each code tab, before the actual commands)

**Review by Serdaro — COMMENTED** [review]
Minor comments.

File: src/docs/asciidoc/getting_started.adoc (hunk 2 above)
Comment (Holmistr) [reply src/docs/asciidoc/getting_started.adoc:24]: This package managers are essential parts of the distributions. I think it's completely safe to not provide the installation instruction links. I have never seen this in any other manual :)

File: src/docs/asciidoc/getting_started.adoc [hunk 3]
```diff
@@ -24,6 +26,30 @@ $ brew tap hazelcast/hz
...
+
+==== Installing Hazelcast with RPM
+
+Run the following commands to install Hazelcast using link:https://rpm.org/[RPM^]:
+
+[source,bash,subs="attributes+"]
+----
+$ wget https://bintray.com/hazelcast/rpm/rpm -O bintray-hazelcast-rpm.repo
+$ mv bintray-hazelcast-rpm.repo /etc/yum.repos.d/
+$ yum install hazelcast
```
[the remark this answers is not in the document]
Comment (alparslanavci) [reply src/docs/asciidoc/getting_started.adoc]: Not sure to mention both. It might be understood as they need to run both?

**Review by Holmistr — APPROVED** [review]

**Review by Serdaro — APPROVED** [review]

File: src/docs/asciidoc/getting_started.adoc (hunk 2 above)
Comment (puzpuzpuz) [reply src/docs/asciidoc/getting_started.adoc:24]: My concern was lack of requirements for the OS. What we had by the time I was writing this is instruction for homebrew, which is only applicable to MacOS, while we have Windows and Linux users.

**Pull request merged**